kvbc/src/reconfiguration_kvbc_handler.cpp

```diff
@@ -266,8 +266,12 @@ bool ReconfigurationHandler::handle(const concord::messages::InstallCommand& com
                                     concord::messages::ReconfigurationResponse&) {
   std::vector<uint8_t> serialized_command;
   concord::messages::serialize(serialized_command, command);
-  auto blockId = persistReconfigurationBlock(
-      serialized_command, bft_seq_num, std::string{kvbc::keyTypes::reconfiguration_install_key}, false);
+  concord::kvbc::categorization::VersionedUpdates ver_updates;
```

Comment:
You can use this https://github.com/vmware/concord-bft/blob/c283616fdc0566afa64c661d7730538c55cddfe7/kvbc/src/reconfiguration_kvbc_handler.cpp#L22. It already adds the epoch.
Also, if you want to start a new epoch, you can add another post reconfiguration handler that adds the new epoch using this https://github.com/vmware/concord-bft/blob/c283616fdc0566afa64c661d7730538c55cddfe7/kvbc/src/reconfiguration_kvbc_handler.cpp#L78